.appveyor.yml

```diff
@@ -9,16 +9,14 @@ branches:
 
 environment:
   matrix:
-    - generator: "Visual Studio 14 2015"
-      toolchain: "Windows"
     - generator: "Visual Studio 15 2017"
```

Comment:
The generator doesn't need to be in a variable and repeated anymore given that it's always the same. You should just get rid of this variable and put the generator name directly in the cmake command below.

Reply:
Done.